OpenFOAM/run.sh

```diff
@@ -16,7 +16,8 @@ if [ -z $EBROOTOPENFOAM ]; then
     exit 1
 fi
 
-export WORKDIR=/tmp/$USER/$$
+# Allow users to define the WORKDIR externally (for example a shared FS for)
```

Comment:
for...? multi-node runs? ;-)

Reply:
Fixed in https://github.com/EESSI/eessi-demo/pull/18/commits/73d9c224428f23ea24936002eb42ce9d68b4ead8